### ISSUE TITLE
Data Labels Misaligned, Depending on Maximum Phase of signal

### ISSUE THREAD

**Issue by MutantPlatypus** [wallento/wavedrompy#13]
With d570c47, data labels are all shifted a little bit based on the most extreme phase shift.  This wasn't an issue before, so I'll roll back through the commits to see which introduced this.
```
{ "signal": [
  { "name": "0",   "wave": "lh===...........", "node": "..a", "data": ["abcd", "1234"], "phase":0},
  { "name": "0.1", "wave": "lh===...........", "node": "..b", "data": ["abcd", "1234"], "phase":-0.1},
  { "name": "0.2", "wave": "lh===...........", "node": "..c", "data": ["abcd", "1234"], "phase":-0.2},
  { "name": "0.3", "wave": "lh===...........", "node": "..d", "data": ["abcd", "1234"], "phase":-0.3},
  { "name": "0.4", "wave": "lh===...........", "node": "..e", "data": ["abcd", "1234"], "phase":-0.4},
  { "name": "0.5", "wave": "lh===...........", "node": "..f", "data": ["abcd", "1234"], "phase":-0.5},
  { "name": "0.6", "wave": "lh===...........", "node": "..g", "data": ["abcd", "1234"], "phase":-0.6},
  { "name": "0.7", "wave": "lh===...........", "node": "..h", "data": ["abcd", "1234"], "phase":-0.7},
  { "name": "0.8", "wave": "lh===...........", "node": "..i", "data": ["abcd", "1234"], "phase":-0.8},
  { "name": "0.9", "wave": "lh===...........", "node": "..j", "data": ["abcd", "1234"], "phase":-0.9},
  { "name": "10",  "wave": "lh===...........", "node": "..k", "data": ["abcd", "1234"], "phase":-10}
  ],
  "edge": [
    "a-b",
    "b-c",
    "c-d",
    "d-e",
    "e-f",
    "f-g",
    "g-h",
    "h-i",
    "i-j",
    "j-k"
  ]
}
```
Renders as:
![image](https://user-images.githubusercontent.com/1389865/63966922-58ff9600-ca6a-11e9-9c7b-527e10e51aa1.png)

To show the dependency on the phase:
![demo+5](https://user-images.githubusercontent.com/1389865/63966262-dd511980-ca68-11e9-8932-acf42addd522.png)
![demo-0](https://user-images.githubusercontent.com/1389865/63966263-dd511980-ca68-11e9-9967-8d93ade65aa0.png)
![demo-5](https://user-images.githubusercontent.com/1389865/63966264-dd511980-ca68-11e9-9fd9-d6ad2ef3235f.png)
![demo-10](https://user-images.githubusercontent.com/1389865/63966265-dd511980-ca68-11e9-8005-ec3a9d70f546.png)

I used the below as a template to render the tests:
```
import cairosvg, wavedrom

svg = wavedrom.render("""
{ "signal": [
  { "name": "+5",  "wave": "l.....h===...........", "node": "........k", "data": ["abcd", "1234"], "phase":5}
  ]
}
""")

svg.saveas('demo+5.svg')

cairosvg.svg2png(svg.tostring(), write_to='.\demo+5.png', output_width=920, output_height=30)
```

**Comment by MutantPlatypus** [comment]
I see you made a commit while I was writing this.  It's still an issue with 7054629fdf9f65613c961c4bb744d7e41d77b43a.

![demo+5](https://user-images.githubusercontent.com/1389865/63967201-dd521900-ca6a-11e9-97cf-73dd78248974.png)
![demo-0](https://user-images.githubusercontent.com/1389865/63967202-dd521900-ca6a-11e9-92f0-171fb411bcab.png)
![demo-5](https://user-images.githubusercontent.com/1389865/63967204-dd521900-ca6a-11e9-851b-c66b3f843ef2.png)
![demo-10](https://user-images.githubusercontent.com/1389865/63967200-dd521900-ca6a-11e9-99f6-074fca10f24a.png)

**Comment by MutantPlatypus** [comment]
Only had to go back one commit.  It works in 55ca3f0dc34ce0e19bc83d0811b460183414962d:
![demo+5](https://user-images.githubusercontent.com/1389865/63967453-74b76c00-ca6b-11e9-9f77-b036f5c1bc57.png)
![demo-0](https://user-images.githubusercontent.com/1389865/63967454-75500280-ca6b-11e9-9759-142b70048a6f.png)
![demo-5](https://user-images.githubusercontent.com/1389865/63967455-75500280-ca6b-11e9-811d-f18c600973d3.png)
![demo-10](https://user-images.githubusercontent.com/1389865/63967456-75500280-ca6b-11e9-9793-6939a6eda675.png)

So d570c47 introduced this.

**Comment by wallento** [comment]
Thanks for reporting and tracking it down!

Fixed in a457b1b
